docs/modules/operation/pages/service-assurance/monitors/JolokiaBeanMonitor.adoc

```diff
@@ -2,12 +2,12 @@
 = JolokiaBeanMonitor
 
 The JolokiaBeanMonitor is a JMX monitor specialized for the use with the link:http://www.jolokia.org[Jolokia framework].
-If it is required to execute a method via _JMX_ or poll an attribute via _JMX_, the _JolokiaBeanMonitor_ can be used.
-It requires a fully installed and configured _Jolokia agent_ to be deployed in the JVM container.
-If required it allows attribute names, paths, and method parameters to be provided additional arguments to the call.
-To determine the status of the service the _JolokiaBeanMonitor_ relies on the output to be matched against a banner.
+If it is required to run a method via JMX or poll an attribute via JMX, the JolokiaBeanMonitor can be used.
```

Comment:
I'm not up on Jolokia to know exactly the right terminology, but this sentence is oddly worded.  Not sure exactly how to fix it.

Reply:
Suggest the following: 

Use the JolokiaBeanMonitor when you need to to run a method via JMX or poll an attribute via JMX.